### PR TITLE
Edit documentations

### LINKED ISSUE #68: Update User Guide
- [x] Fatin: 1, 2
- [x] YiJing: 3
- [x] Sam: 4.1 - 4.2
- [x] Gabe: 4.3
- [x] Rai: 4.4 - 4.7

### LINKED ISSUE #69: Update Developer Guide
- [x] A: YiJing
- [x] B: Rai
- [x] C: Gabe, Fatin
- [x] D, E: Sam

### PR DESCRIPTION
**Pull request by aakanksha-rai**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -74,88 +74,100 @@ Examples:
...
-=== Editing a person : `edit`
+=== Student Management
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+==== Enrolling a student in a class : `enroll`
+
+Enrolls a student in a class in the program. +
+
+Format: `enroll MATRIC_NUMBER CLASS_CODE MOD_CODE'
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: The last quote mark doesn't match!!

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -74,88 +74,100 @@ Examples:
...
+Removes a student from a class in the program. +
+Format: `kick MATRIC_NUMBER CLASS_CODE MOD_CODE`
 
 ****
-* The search is case insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* Removes the person with the given matriculation number from the class given by the class code of the module given by the module code.
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Maybe could be split into smaller dot points.
eg.
- Removes the person with the given matriculation number from the specified class
- The class should belong to the module with the MODULE_CODE
- The class should already in the program with the existing CLASS_CODE

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -74,88 +74,100 @@ Examples:
...
+
+Format: `enroll MATRIC_NUMBER CLASS_CODE MOD_CODE'
 
 ****
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
+* Enrolls the person with the given matriculation number in the class given by the class code of the module given by the module code.
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Maybe could be split into smaller dot points.
eg.
- Enrolls the person with the given matriculation number in the specified class
- The class should belong to the module with the MODULE_CODE
- The class should already in the program with the existing CLASS_CODE

File: docs/UserGuide.adoc [hunk 4]
```diff
@@ -74,88 +74,100 @@ Examples:
...
-`delete 1` +
-Deletes the 1st person in the results of the `find` command.
+* `done 25` +
+Marks the session with the unique session id of 25 as done.
+
+==== Labeling a session as a makeup : `makeup`
+
+Labels a session as a makeup session. If the session is claimable, it will appear
+ as a new claim in the TSS view.+
+Format: `makeup SESSION_UID`
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Missing newline!

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -74,88 +74,100 @@ Examples:
...
 
-// tag::dataencryption[]
-=== Encrypting data files `[coming in v2.0]`
+*rate:* Adjusts the hourly rate. +
+Sets the hourly rate for the total income and claim computation.
+
+Format: `set rate AMOUNT`
+* AMOUNT is the amount you want to change the hourly rate to.
+* To specify in exact dollars, you can write it as just the number (example: 20).
+* To specify in exact dollars and cents, write it as a decimal up to 2 decimal places (example: 20.05).
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc:205]: The dot points are missing!

**Review by PotatoCombat — COMMENTED** [review]
Just need to fix some formatting issues and you're done!

File: docs/UserGuide.adoc [hunk 6]
```diff
@@ -74,88 +74,106 @@ Examples:
...
+
+Format: `enroll MATRIC_NUMBER CLASS_CODE MOD_CODE`
 
 ****
-* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person's tags by typing `t/` without specifying any tags after it.
+* Enrolls the person with the given matriculation number in the specified class.
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Sorry, can change person to student?

File: docs/UserGuide.adoc [hunk 7]
```diff
@@ -74,88 +74,106 @@ Examples:
...
+Removes a student from a class in the program. +
+Format: `kick MATRIC_NUMBER CLASS_CODE MOD_CODE`
 
 ****
-* The search is case insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search). e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* Removes the person with the given matriculation number from the specified class.
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Same... can change person to student?

File: docs/UserGuide.adoc [hunk 8]
```diff
@@ -74,88 +74,106 @@ Examples:
...
+* `enroll A01234567J G06 CS2101` +
+Enrolls the student with the matriculation number A01234567J in tutorial G06 of module CS2101.
 
-=== Locating persons by name: `find`
+==== Removing a student from a class : `kick`
 
-Finds persons whose names contain any of the given keywords. +
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Removes a student from a class in the program. +
+Format: `kick MATRIC_NUMBER CLASS_CODE MOD_CODE`
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Could you add a newline between the description and format?

File: docs/UserGuide.adoc [hunk 9]
```diff
@@ -74,88 +74,106 @@ Examples:
...
-=== Editing a person : `edit`
+=== Student Management
 
-Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+==== Enrolling a student in a class : `enroll`
+
+Enrolls a student in a class in the program. +
+
+Format: `enroll MATRIC_NUMBER CLASS_CODE MOD_CODE`
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc:124]: This spacing is good!

File: docs/UserGuide.adoc [hunk 10]
```diff
@@ -74,88 +74,106 @@ Examples:
...
 
-// tag::delete[]
-=== Deleting a person : `delete`
+==== Labelling a session as completed : `done`
 
-Deletes the specified person from the address book. +
-Format: `delete INDEX`
+Labels a session as done. If the session is claimable, it will appear as
+a new claim in the TSS view.+
+Format: `done SESSION_UID'
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Spacing please!

Also the newline didn't show properly

File: docs/UserGuide.adoc [hunk 11]
```diff
@@ -74,88 +74,106 @@ Examples:
...
-`delete 1` +
-Deletes the 1st person in the results of the `find` command.
+* `done 25` +
+Marks the session with the unique session id of 25 as done.
+
+==== Labeling a session as a makeup : `makeup`
+
+Labels a session as a makeup session. If the session is claimable, it will appear
+ as a new claim in the TSS view. +
+Format: `makeup SESSION_UID`
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Spacing please!

File: docs/UserGuide.adoc [hunk 12]
```diff
@@ -74,88 +74,106 @@ Examples:
...
-Format: `clear`
+=== Settings
 
-=== Exiting the program : `exit`
+==== Adjusting the settings : `set`
 
-Exits the program. +
-Format: `exit`
+Adjusts the chosen settings. +
+Format: `set SETTING`
```
Comment (PotatoCombat) [inline docs/UserGuide.adoc]: Spacing please!

**Review by PotatoCombat — COMMENTED** [review]
Just need some spacing between format and description!

**Review by PotatoCombat — APPROVED** [review]
All good to merge!

File: docs/UserGuide.adoc [hunk 13]
```diff
@@ -74,88 +74,110 @@ Examples:
...
+=== Session Management
 
-Deletes the specified person from the address book. +
-Format: `delete INDEX`
+==== Labelling a session as completed : `done`
+
+Labels a session as done.
+If the session is claimable, it will appear as a new claim in the TSS view.
+
+Format: `done SESSION_UID'
```
Comment (fatin99) [inline docs/UserGuide.adoc]: should be ` instead of ' at the end of the line

**Review by fatin99 — COMMENTED** [review]
just one small error

**Review by fatin99 — APPROVED** [review]
all good :)

**Review by PotatoCombat — APPROVED** [review]

**Comment by PotatoCombat** [comment]
All good to merge!

**Pull request merged**